docs/advanced/experimental/indexers-benchmark.md

```diff
@@ -0,0 +1,60 @@
+# Indexers Benchmark
+
+In this page we show the performance capabilities of the different Indexers available in Jina Hub.
+
+## Results
+
+In the below graphs you can see the performance, in terms of speed and evaluation.
+Speed is reported for query time, in total seconds.
+Evaluation is provided in both precision and recall, out of the `top_k` 100.
+The tables are sorted by `time_search`.
+
+```{include} bench/gist.md
+```
+
+```{include} bench/sift.md
+```
+
+```{include} bench/sift1bil.md
+```
```

Comment:
In the tables inside, precision and recall is always identical.

Reply:
Ah the reason is, because they are both: from the top 100 and in the top 100. :D This looks a bit weird.

Reply:
This might be the reason, they don't care about precision in the [ann benchmark](http://ann-benchmarks.com/)

Reply:
Yeah, we should remove it then. The formula seems correct me to in the [code](https://github.com/jina-ai/benchmark_indexers/blob/57276a44267e193127b65954b07fa6f529e0bd23/src/evaluator.py#L20)

Reply:
If you take 100 correct Documents and check how many of them are in the top 100, precision and recall are the same. Because it is both `number of correct documents / 100`.